.github/workflows/run-integ-tests.yaml

```diff
@@ -139,6 +139,13 @@ jobs:
       matrix:
         app_to_test: ${{ fromJson(needs.list-apps.outputs.to_test) }}
         mode: [fresh, upgrade]
+        exclude:      
```

Comment:
Can you file issues for the ignored tests (or just one, if they're the same problem) and add a reference to it as comments to the ignore?

Reply:
yup will do